lib/lightning/ai_assistant/ai_assistant.ex

```diff
@@ -43,7 +45,15 @@ defmodule Lightning.AiAssistant do
 
   @spec get_session!(Ecto.UUID.t()) :: ChatSession.t()
   def get_session!(id) do
-    ChatSession |> Repo.get!(id) |> Repo.preload(messages: :user)
+    message_query =
+      from(m in ChatMessage,
+        where: m.status != :cancelled,
```

Comment:
👍

Reply:
Yes @jyeshe, we return the error ones so that they can be retried.